spark-bigquery-connector-common/src/main/java/com/google/cloud/spark/bigquery/direct/BigQueryRDDFactory.java

```diff
@@ -95,7 +95,7 @@ public RDD<InternalRow> buildScanFromSQL(String sql) {
 
     log.info(
         "Querying table {}, requiredColumns=[{}]",
-        actualTable.getFriendlyName(),
+        actualTable.getTableId().getTable(),
```

Comment:
`actualTable.getFriendlyName()` return null. `actualTable.getTableId().getTable()` returns the correct name